src/lib/constants/deployment-addresses.ts

```diff
@@ -0,0 +1,7 @@
+import { ChainId } from '../../utils';
+
+export const DEPLOYMENT_ADDRESSES = {
+    [ChainId.MAINNET]: {
+        weightedFactoryV4: '0x897888115ada5773e02aa29f775430bfb5f34c51',
```

Comment:
This is duplicated from the constants file. Can we remove it?

Reply:
Yeah, of course